src/core/dbus-mount.c

```diff
@@ -51,6 +51,7 @@ const sd_bus_vtable bus_mount_vtable[] = {
         SD_BUS_PROPERTY("SloppyOptions", "b", bus_property_get_bool, offsetof(Mount, sloppy_options), SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("LazyUnmount", "b", bus_property_get_bool, offsetof(Mount, lazy_unmount), SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("ForceUnmount", "b", bus_property_get_bool, offsetof(Mount, force_unmount), SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("NoRemountRO", "b", bus_property_get_bool, offsetof(Mount, no_remount_ro), SD_BUS_VTABLE_PROPERTY_CONST),
```

Comment:
Hmm, we typically don't abbreviate read-only as RO in prop names

Also, please do not expose "negative" properties. i.e. `NoRemountRO=no` is just confusing, because you negate a negation here.

Hence, let's reverse the property. i.e. call it ReadOnlyFallback= or so, to clearly indicate what this is about, use no unneded abbreviation and no negation

Reply:
Yeah, my initial choice was `ReadWriteOnly=`. The fstab equivalent would be `x-systemd.read-write-only`.

I still prefer `ReadWriteOnly=`...

Reply:
sounds good to me. Or maybe `StrictlyReadWrite=`?

Reply:
Went with `ReadWriteOnly=` unit setting, and `x-systemd.rw-only` fstab option